src/Files.Uwp/Themes/Discord.xaml

```diff
@@ -1,53 +0,0 @@
-
-<ResourceDictionary
```

Comment:
Why was this removed?

Reply:
All custom themes removed, some were giving compile errors (maybe related to mica/acrylic)

Reply:
I'd like to keep them in the repo, at the very least we can exlude them from the project for now.

Reply:
Custom themes restore. Exceptions are the acrylic-glass-mica imitations themes where I had to remove the BackgroundSource property from the Acrylic brushes (not supported)

Reply:
We should definitely remove the acrylic themes.